src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1045,3 +1045,24 @@ function dorogovtsev_mendes(n::Integer; seed::Int=-1)
     end
     return g
 end
+
+"""
+    random_orientation_dag(n)
+
+Generate a random oriented acyclical digraph. The function takes in a simple/
+graph as an argument. 
+"""
+function random_orientation_dag(g::SimpleGraph{T}) where T <: Integer
+    nv = length(g.fadjlist)
+    tnv = T(nv)
+    order = randperm(tnv)
+    g2 = SimpleDiGraph(tnv) #initialize empty adj lists for each node; both f and b
+    for i in (1 : tnv)
+        for j in 1:length(g.fadjlist[i])
```

Comment:
```suggestion
        for j in eachindex(g.fadjlist[i])
```

Reply:
or, even better, `for j in 1:outdegree(g, i)` - don't use internal structures unless you have to.

Reply:
edited

Reply:
done